tests/integration/restricted/test.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -ex
+
+RANDOM_CLUSTER_NAME_SUFFIX=`cat /dev/urandom | tr -cd 'a-f0-9' | head -c 7`
+
+CLUSTER_NAME="ds-test-restricted-${RANDOM_CLUSTER_NAME_SUFFIX}"
```

Comment:
may be useful to use a buildkite identifier if we have one? would be easier to link between the build and the cluster.

Reply:
ok, will try